tests/jupyterhub_api/test_stop_session.py

```diff
@@ -0,0 +1,85 @@
+
```

Comment:
One could also check if the container has been removed, likely re-using the 'check_container' function in 'test_check_session.py' that should be moved somewhere else to avoid repeated code.
How does this work?
Is the container immediately stopped or there should be a wait time?

Reply:
Yes, we could use `check_container` function, I'll add it here, the API sends a request to stop the server and after a particular wait time (which is hardly noticeable maybe 1-2 seconds) the server is stopped.